.github/workflows/deploy_preview.yml

```
@@ -24,11 +26,73 @@ jobs:
 
       - name: Deploy to Netlify
         uses: nwtgck/actions-netlify@v1.2
+        id: deploy
         with:
           publish-dir: './build'
+          alias: ${{ github.event.number }}
           deploy-message: '#${{ github.event.pull_request.number }} - ${{ github.event.pull_request.title }}'
           github-token: ${{ secrets.GITHUB_TOKEN }}
+          enable-pull-request-comment: false
         env:
           NETLIFY_AUTH_TOKEN: ${{ secrets.NETLIFY_AUTH_TOKEN }}
           NETLIFY_SITE_ID: ${{ secrets.NETLIFY_SITE_ID }}
         timeout-minutes: 1
+
+  deploy-storybook-preview:
+    name: Build and Deploy Storybook Preview on Netlify
+    runs-on: ubuntu-latest
+    environment: staging
+    outputs:
+      deploy-url: ${{ steps.deploy.outputs.deploy-url }}
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+        with:
+          access_token: ${{ github.token }}
+
+      - name: Checkout
+        uses: actions/checkout@v2
```

```suggestion
        uses: actions/checkout@v3
```
Can you pump up the latest version here and in other places?

---

Updated all actions [18a1e0c](https://github.com/rosle/react-nimble-survey/pull/23/commits/18a1e0cc7e4781b90b18d45a471a8ad1e6db2a5f) 👍